app/controllers/poems_controller.rb

```diff
@@ -4,17 +4,17 @@ def new
   end
 
   def create
-    @poem = Poem.create(poem_params)
-
+    @poem = current_user.poems.create(poem_params)
+    
     respond_with @poem
   end
 
   def show
-    @poem = Poem.find(params[:id])
+    @poem = Poem.find(params[:id]) || Poem.new
```

Comment:
`find` will raise an error if nothing is found so the `|| Poem.new` will never trigger.